docker/start.sh

```diff
@@ -5,10 +5,29 @@ if [ -z "$REINDEX" ]; then
 	REINDEX=10
 fi
 
+if [ "${OPENGROK_WEBAPP_CONTEXT}" = "/" ]; then
```

Comment:
If `OPENGROK_WEBAPP_CONTEXT` is empty or contains whitespace, should it be reset earlier to `/` to benefit this conditional and later usage?

Reply:
Sounds good, but we'll have to take a look at format restrictions for the variable later anyways (leading slash, trailing slash, etc.).